### PR TITLE
FTT-11534 update applicationinsights to latest version

### PR DESCRIPTION
**Pull request by ThomasPritchard**
- Bumped `applicationinsights` to latest version 2.1.5
- Ran `npm audit fix` to clean the `package-lock.json`
- Removed dotenv error throwing to prevent crashing in other components that don't use .env files

### CODE REVIEW

File: src/config/index.ts [hunk 1]
```diff
@@ -1,13 +1,3 @@
-import dotenv from 'dotenv';
-
-if (!process.env.APPINSIGHTS_INSTRUMENTATIONKEY) {
-  const result = dotenv.config();
```
Comment (MohammedFoysal) [inline src/config/index.ts:4]: I think we should still let dotenv work in the library, just swallow the error instead of throwing

**Review by MohammedFoysal — CHANGES_REQUESTED** [review]

**Review by MohammedFoysal — APPROVED** [review]

**Review by premghinde — APPROVED** [review]

File: src/config/index.ts [hunk 2]
```diff
@@ -1,11 +1,9 @@
 import dotenv from 'dotenv';
 
+dotenv.config();
```
Comment (Cavalier07) [inline src/config/index.ts:3]: This should neverbe used in the code

Comment (Cavalier07) [reply src/config/index.ts:3]: Should only be loaded via npm scripts like we do in the booking app

Comment (ThomasPritchard) [reply src/config/index.ts]: done

File: package.json [hunk 3]
```diff
@@ -31,7 +31,7 @@
     "typescript": "3.9.7"
   },
   "dependencies": {
-    "applicationinsights": "1.8.7",
+    "applicationinsights": "2.1.5",
     "dotenv": "8.2.0",
```
Comment (premghinde) [inline package.json:35]: delete ;)

**Pull request merged**